cmds/core/dd/dd.go

```diff
@@ -401,7 +401,9 @@ func run(stdin io.Reader, stdout io.WriteSeeker, stderr io.Writer, name string,
 	// convert the arguments and then run flag.Parse. Gross, but hey, it
 	// works.
 	args = convertArgs(args)
-	f.Parse(args)
+	if err := f.Parse(args); err != nil {
+		return fmt.Errorf("run: %w", err)
```

Comment:
same here. Plus, in one command, it's
error parsing flags
and here it's
run

for sure, users of dd will have no idea what the run: part of this text means. Is the error returned by f.Parse good enough that we don't need extra text? We could then just
return err